arch/risc-v/include/inttypes.h

```diff
@@ -29,7 +29,18 @@
  * Pre-processor Definitions
  ****************************************************************************/
 
-#if defined(CONFIG_ARCH_RV64)
+#if defined(CONFIG_ARCH_RV64ILP32)
```

Comment:
should we reuse the same definition as CONFIG_ARCH_RV32? e..g.:
```
#define _PRI32PREFIX "l"
```

Reply:
I am unsure, actually the 32 bit definitions looks a little strange to me as `int` is 32-bit thus no modifiers is needed.